migrations/20190820204114-create-food.js

```diff
@@ -9,7 +9,8 @@ module.exports = {
         type: Sequelize.INTEGER
       },
       name: {
-        type: Sequelize.STRING
+        type: Sequelize.STRING,
+        unique: true
```

Comment:
Need to revert this file back to the original -- never edit a migration file that's already been run on a computer other than your own (especially if it's already run in production).